shell/initialize/client.js

```diff
@@ -660,6 +694,9 @@ const noopData = () => {
 const noopFetch = () => {};
 
 // Special hot reload with asyncData(context)
+// still don't fully understand all of these functions, but doesn't vue offer hot reload?
```

Comment:
asyncData is a nuxt page load thing. this is them tying that mechanism into hot reload